### ISSUE TITLE
Readme is confusing

### ISSUE THREAD

**Issue by andriy-f** [Vheissu/aurelia-configuration#81]
Readme is confusing. For example: 
code block just after "Add plugin to your app's main.js:" and next code block are different, hard to follow what was changed. And ".plugin('aurelia-configuration')" block has been removed

**Comment by Vheissu** [comment]
@johnsmith9264 Is the Readme still confusing for you? Some improvements were made.

**Comment by johnsmith9264** [comment]
there were two blocks of code (one **Add plugin to your app's main.js or main.ts file:**, another **Use the plugin to set configuration in your app's main.js or main.ts file:**). I was confused that these blocks described same function (aurelia initialization) and second block was missing addition of usage `.plugin('aurelia-configuration');`. 

But now I get that first code fragment was showing how to **add** plugin and second how to **use** it. Maybe make fragments which were added bold?
(like
<pre>
standard operator
<b>example operator</b>
standard operator 2
</pre>
